privaterelay/templates/profile.html

```diff
@@ -54,11 +54,11 @@
   {% endif %}
   <div class="main-list flx flx-col jst-cntr">
   {% if messages %}
-    <div class="messages profile-messages hidden">
+    <div class="messages hidden">
         {% for message in messages %}
         <div{% if message.tags %} class="{{ message.tags }} js-notification" {% else %} class="" {% endif %}>
           <span>{{ message }}</span>
-          <button class="flx center-value dismiss js-dismiss">
+          <button class="dismiss">
```

Comment:
By removing the `js-dimiss` class, I would expect something to break? Note that it's referenced [in the app.js file](https://github.com/mozilla/fx-private-relay/blob/exit-btn-color/static/js/app.js#L269-L271). You may need to keep/add back the `js-dimiss` class.

Reply:
Haha yes its supposed to be green, I think I was testing the `error` state by changing the class in Inspector. The thing is... nothing broke (at least not to my knowledge) when I removed `js-dismiss`, so should I just keep it regardless?